tests/unit_tests/handlers/test_delivery_organise_handlers.py

```diff
@@ -0,0 +1,163 @@
+from tornado.testing import *
+from tornado.web import Application
+
+import tempfile
+import mock
+import pathlib
+
+from delivery.app import routes
+
+from tests.test_utils import DummyConfig
+
+class TestOrganiseHandlers(AsyncHTTPTestCase):
+    API_BASE = "/api/1.0"
+
+    def get_app(self):
+        self.organise_service = mock.MagicMock()
+        self.config = DummyConfig()
+
+        self.project = "CD-1234"
+        self.project_path = (
+            pathlib.Path(self.config["general_project_directory"]) / self.project
+        )
+        self.project_path.mkdir()
+
+        self.project_config_path = (
+            pathlib.Path(self.config["organise_config_dir"]) / "sarek.yml"
+        )
+        self.project_config_path.touch()
+
+        self.runfolder = "230721_A000_BHXX"
+        self.runfolder_path = (
+            pathlib.Path(self.config["runfolder_directory"]) / self.runfolder
+        )
+        self.runfolder_path.mkdir()
+
+        self.runfolder_config_path = (
+            pathlib.Path(self.config["organise_config_dir"]) / "runfolder.yml"
+        )
+        self.runfolder_config_path.touch()
+
+        return Application(
+            routes(
+                config=self.config,
+                organise_service=self.organise_service,
+                ))
+
+    def test_project_analysis_handler(self):
+        analysis = "sarek"
+
+        response = self.fetch(
+            f"{self.API_BASE}/organise/delivery/project/{analysis}/{self.project}",
+            method="POST", body="",
+        )
+
+        self.assertEqual(response.code, 200)
+        self.organise_service.organise_with_config.assert_called_with(
+            str(self.project_config_path), str(self.project_path))
+
+    def test_project_analysis_handler_missing_project(self):
+        analysis = "sarek"
+        project = "AB-4567"
+
+        response = self.fetch(
+            f"{self.API_BASE}/organise/delivery/project/{analysis}/{project}",
+            method="POST", body="",
+        )
+
+        self.assertEqual(response.code, 404)
+
+    def test_project_analysis_handler_missing_config_file(self):
+        analysis = "rnaseq"
```

Comment:
super nitpick: might be a good idea to use a dummy value instead of a potentially valid analysis name